atom/common/crash_reporter/crash_reporter_mac.mm

```diff
@@ -67,7 +67,7 @@
   simple_string_dictionary_.reset(new crashpad::SimpleStringDictionary());
   crashpad_info->set_simple_annotations(simple_string_dictionary_.get());
 
-  SetCrashKeyValue("prod", CHROMIUM_SHORT_NAME);
+  SetCrashKeyValue("prod", ATOM_PRODUCT_NAME);
```

Comment:
using CHROMIUM_SHORT_NAME is intentional and should return Brave. If it doesn't that is a an issue we need to look at because it's used in other places